libraries/RC_Channel/RC_Channel.h

```diff
@@ -219,16 +219,17 @@ class RC_Channel {
         // also, if you add an option >255, you will need to fix duplicate_options_exist
 
         // options 150-199 continue user rc switch options
-        CRUISE =             150,  ///CRUISE mode
+        CRUISE =             150,  // CRUISE mode
         TURTLE =             151,  // Turtle mode - flip over after crash
         SIMPLE_HEADING_RESET = 152, // reset simple mode refernce heading to current
         ARMDISARM =          153, // arm or disarm vehicle
         ARMDISARM_AIRMODE =  154, // arm or disarm vehicle enabling airmode
         TRIM_TO_CURRENT_SERVO_RC = 155, // trim to current servo and RC
         TORQEEDO_CLEAR_ERR = 156, // clear torqeedo error
-        EMERGENCY_LANDING_EN = 157, //Force long FS action to FBWA for landing out of range
+        EMERGENCY_LANDING_EN = 157, // Force long FS action to FBWA for landing out of range
+        HEADING_HOLD =       158,  // select heading hold or rate mode for yaw controller
```

Comment:
overall this is confusing....the function is actually HEADING_HOLD_ENABLE   ie high (the active state for most switches) is NOT disabling the HEADING HOLD....the description should be "enable heading hold mode (use only rate mode for yaw controller)...but having said that I think the naming is still backwards....the defualt, no switch state, is HEADING HOLD...and what the switch is doing is DISABLING Heading Hold from the normal state....so I think that the sw function shoud be named HEADING_HOLD_DISABLE  with high state making plane.heading_hold_disable = true and the description "Disable Heading Hold (ie only rate mode for Yaw)"

is this only valid in ACRO mode? then the description should be "Disable ACRO Heading Hold (ie only rate mode for Yaw in ACRO)"

Reply:
good points, I had the implementation backwards too, at first :)

Reply:
sorry, still think that since heading hold is the default and the switch is adding the ability to DISABLE it, the switch should be called HEADING HOLD DISABLE and a high switch  is plane.heading_hold_disable = true and its description is "Disable Heading Hold in ACRO mode (ie only rate mode for Yaw)"

Reply:
@Hwurzburg I changed the default state to HH off (heading_hold_disable= true)
So if you want to use heading hold mode, you have to assign the RC option.
I want HH off for planes unless I need some extra help in knife-edge flight, and I definitely want it off as I exit knife-edge.  Also try doing a quick 180 with and without HH on and see which you prefer.